includes/admin/views/omise-page-settings.php

```diff
@@ -16,7 +34,26 @@
 		);
 		?>
 	</p>
+
 	<form method="POST">
+		<!-- Section: account information -->
+		<?php if ( $settings['account_email'] ) : ?>
+			<hr />
+			<table class="form-table">
+				<tbody>
+					<tr>
+						<th scope="row"><label><?php _e( 'Account status', 'omise' ); ?></label></th>
```

Comment:
I was choosing `Account status` to support a case where that particular account get suspended then, instead of showing "connected", we can change its status to `blocked` or `suspended`. 

The output at this moment is:
> Account status: Connected: _user@omise.co (TH)_

The improvement will be
> Account status: Suspended: _user@omise.co (TH)_

Reply:
I understand.